git-extra/astextplain

```diff
@@ -8,6 +8,9 @@ fi
 
 # XXX output encoding (UTF-8) hardcoded
 case "$1" in
+	*.ods | *.ODS | *odf |*.ODF | *odt | *.ODT)
+		odf2txt "$1" || cat "$1"
```

Comment:
Oh, and there really is a need to use [`_realname`, `pkgbase` and `pkgname`](https://github.com/git-for-windows/MINGW-packages/commit/898f5ed4bd2c6f5addd60aabb977051d97d0350f#diff-312c6a788fc5432403915ac0aa4090b6R3), and you most likely will want to change the dependency to `${MINGW_PACKAGE_PREFIX}-zlib`. Otherwise you would depend on the *MSYS2* version of zlib, which would be totally useless (you cannot link a MINGW executable to an MSYS2 library).